test/verify/machineslib.py

```diff
@@ -1298,6 +1298,61 @@ def installWithErrorTest(dialog):
                                          storage_pool="No Storage",
                                          start_vm=True,))
 
+        # Test create VM with disk of type "block"
+        self.machine.add_disk("50M", serial="DISK1")
+        wait(lambda: self.machine.execute("ls -l /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1"))
+        cmds = [
+            "parted /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1 mklabel gpt",
+            "virsh pool-define-as poolDisk disk - - /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1 - /tmp/poolDiskImages",
+            "virsh pool-build poolDisk --overwrite",
+            "virsh pool-start poolDisk",
+            "virsh vol-create-as poolDisk sda1 1024"
+        ]
+        self.machine.execute(" && ".join(cmds))
+
+        self.browser.reload()
```

Comment:
The above commands are emitting signal and the storage pool list is updated by them.
Thus there is not need to call browser.reload() here.
I guess there reason you put it is not gain enough time for the state to actually get updated.
A better approach for that would be to monitor the number of active pools in the Cards on the top of the page.

Reply:
Now when I see all the virsh related test failures, I remember why I have done it this way. This functionality is meant to work also on virsh provider, so the tests should also be for virsh provider. But pool Cards are visible only when you have DBus provider right?

Reply:
We could parse the output of some virsh command.

Reply:
Oh right, but it doesn't garrantee you more than the reload that the UI will be actually updated.
This smells that it can become a race but's let's see in practice.

Reply:
So do you agree more with checking virsh? Or reloading page? Or both?
Or some timeout?

Reply:
Any if these is going to be possible race condition, but we really don't have better option here. You can live it with the reload if it works.